hip_data_tools/etl/adwords_to_s3.py

```diff
@@ -168,6 +170,7 @@ class AdWordsReportToS3Settings:
     target_key_prefix: str
     target_file_prefix: Optional[str]
     target_connection_settings: AwsConnectionSettings
+    transformation_field_type_mask: Optional[Dict[str, Type]]
```

Comment:
instead of python types, make it so the type required ia s numpy type, which can be directly applied to the dataframe:
https://numpy.org/doc/stable/user/basics.types.html

Reply:
done